### PR TITLE
Cleanup core imports

### PR DESCRIPTION
**Pull request by josephco**
Refactor core imports to stop generating output and group related styles together

### CODE REVIEW

File: src/layout/_base.scss [hunk 1]
```diff
@@ -46,3 +46,8 @@ body {
     box-sizing: inherit;
   }
 }
+
+// Disable 300ms delay on touch devices
+.no-touch-delay {
+  touch-action: manipulation;
+}
```
Comment (misscs) [inline src/layout/_base.scss:53]: Ooo, like this location for this code. +1

File: src/layout/_helper_classes.scss [hunk 2]
```diff
@@ -1,81 +1,38 @@
...
   margin-left: auto;
   margin-right: auto;
 }
 
-
 .-hoisted {
   @include respond-at-and-above(1200px) {
     margin-top: -40px;
   }
 }
```
Comment (misscs) [inline src/layout/_helper_classes.scss:43]: Let's go ahead and remove lines 29-38. It is old code and we don't have these concepts in core yet. may as well keep this lean.

File: src/layout/_helper_classes.scss [hunk 3]
```diff
@@ -1,81 +1,38 @@
-// Layout
-//
-// 1. Spree-auth-devise hax which I hate
-//
-// @@@ TODO
-// This file should just be positioning/layout helper classes
+// Layout helper classes
```
Comment (misscs) [inline src/layout/_helper_classes.scss]: Please add file level docs :)

File: src/layout/_helper_classes.scss (hunk 2 above)
Comment (josephco) [inline src/layout/_helper_classes.scss:43]: Added a todo to clean this up when we get to static storefront stuff

**Pull request merged**